.circleci/config.yml

```diff
@@ -1,46 +1,51 @@
-# Python CircleCI 2.0 configuration file
+# Python CircleCI 2.1 configuration file
 #
 # Check https://circleci.com/docs/2.0/language-python/ for more details
 #
 
-version: 2
+version: 2.1
 workflows:
-  version: 2
+  version: 2.1
   test:
     jobs:
-      - test-3.6
+      - test:
+          matrix:
+            parameters:
+              python_version: [ "3.5", "3.8"]
 jobs:
-  test-3.6:
+  test:
+    parameters:
+      python_version:
+        type: string
+    working_directory: ~/repo<< parameters.python_version >>
     docker:
-      # specify the version you desire here
-      # use `-browsers` prefix for selenium tests, e.g. `3.6.1-browsers`
-      - image: circleci/python:3.6.5
-    working_directory: ~/repo
-
+      - image: 'circleci/python:<< parameters.python_version >>'
     steps:
       - checkout
-
-      # Download and cache dependencies
       - restore_cache:
           keys:
           - v1-dependencies-{{ checksum "requirements/test.txt" }}
-          # fallback to using the latest cache if no exact match is found
-          - v1-dependencies-
-
       - run:
-          name: activate virtualenv and install dependencies.
+          name: Activating virtualenv and installing dependencies.
           command: |
             test -d venv || virtualenv -p python3 ./venv
             . venv/bin/activate
             pip install -r requirements/test.txt
+            pip install -r requirements/quality.txt
+            pip install -r requirements/tox.txt
```

Comment:
I think you can use: install-dev-dependencies here, can't you? @ivanvgh

Reply:
Yes, you are right, changed to use the make command.

Reply:
Yes, you are right, changed to use the make command.